### DISCUSSION TITLE
AttributeError: 'CustomMBartDecoder' object has no attribute 'embed_scale' 这个是因为啥？

### DISCUSSION THREAD

**Discussion by zhangheng2604** [magicpdf/Magic-PDF#453] [Q&A]
AttributeError: 'CustomMBartDecoder' object has no attribute 'embed_scale'

**Comment by TechnicolorGUO** [comment]
可能是transformers版本的问题 重新装一下magic-pdf就好了

**Comment by yangzengwu** [comment]
确实是transformers的问题。降了版本就可以了
pip install transformers==4.40.0
